index.bs

```diff
@@ -12748,7 +12722,7 @@ The \[[Prototype]] [=internal slot=] of an [=asynchronous iterator prototype obj
 
     1.  If |ongoingPromise| is not null, then:
         1.  Let |afterOngoingPromiseCapability| be [=!=] [$NewPromiseCapability$]({{%Promise%}}).
-        1.  Let |onSettled| be [=!=] [$CreateBuiltinFunction$](|nextSteps|, « »).
+        1.  Let |onSettled| be [=!=] [$CreateBuiltinFunction$](|nextSteps|, 0, "", « »).
```

Comment:
Are these observable? Should the length be one like the others?

Reply:
> Are these observable?

I don’t think so, since it’s passed directly to [PerformPromiseThen](https://tc39.es/ecma262/#sec-performpromisethen), rather than going through <code>[%Promise.prototype%.then](https://tc39.es/ecma262/#sec-promise.prototype.then)</code>.

----

> Should the length be one like the others?

Well, <var>nextSteps</var> doesn’t take any parameters, so it’s similar to:
```js
const nextSteps = () => { ... };
```
Which has a length of `0`.